src/Packetery/Module/Options/FeatureFlagManager.php

```diff
@@ -113,52 +135,41 @@ private function fetchFlags(): array {
 	 * @throws Exception From DateTimeImmutable.
 	 */
 	private function getFlags(): array {
-		static $flags;
-
-		if ( ! isset( $flags ) ) {
-			$flags = get_option( self::FLAGS_OPTION_ID );
+		if ( ! isset( self::$flags ) || null === self::$flags ) {
+			self::$flags = $this->wpAdapter->getOption( self::FLAGS_OPTION_ID );
 		}
 
-		if ( true === get_option( self::DISABLED_DUE_ERRORS_OPTION_ID ) ) {
-			return $flags ? $flags : [];
+		if ( true === $this->wpAdapter->getOption( self::DISABLED_DUE_ERRORS_OPTION_ID ) ) {
+			return self::$flags ? self::$flags : [];
```

Comment:
return self::$flags ?: [];

Reply:
`Using short ternaries is not allowed | (WordPress.PHP.DisallowShortTernary.Found)`

Reply:
ok, thx.